packages/office-addin-manifest/src/cli.ts

```diff
@@ -22,6 +22,13 @@ commander
 
 commander.command("validate <manifest-path>").action(commands.validate);
 
+commander
+  .command("export")
+  .option("-m, --manifest <manfest-path>", "Specify the location of the manifest file.  Default is './manifest.json'")
+  .option("-a, --assets <assets-folder-path>", "Specify the location of the assets folder.  Default is './assets'")
+  .option("-o, --output <output-path>", "Specify where to save the package.  Default is './package/manifest.zip'")
```

Comment:
Can we put just `manifest.zip` as default?
I feel having the file in the same directory is more intuitive.

Reply:
I wasn't sure where the best place for the output should be by default.  Some thoughts include

- root as you suggested
- temp folder
- folder in the project we could mark to be ignored (so it doesn't show up as a new file for the project).

Haven't found something I'm particularly drawn to.

Reply:
Well, I'm drawn to being in the same directory the command was called, so `./manifest.zip`.
We can also put it to be ignored as well, even if we don't know the file:
Something like `*manifest.zip` should work.

Reply:
Or maybe I put it in the same location the manifest file came from instead of specifying a specific default location.  This would be just ./manifest.zip for the default argument.

I'd like to use the .gitignore . . . however that only works for the default case.  If anything else is specified then it won't do any good.

Reply:
Yeah, `.gitignore` would only solve for the default case, but I think it is better than nothing.

Reply:
Actually, .gitignore would go in the template project not the script package here